### ISSUE TITLE
Suggestion: Layout color

### ISSUE THREAD

**Issue by nedark** [Codiad/Codiad#785]
Although I can change editor color, I cant change the layout black background. I would like a light layout, however it looks that themes only affect to editor. And I will only say one thing about a light editor and black layout: its awful. 
It least, an option to switch from a dark and light layouts would be appreciated. I think that would be enough since all themes are quite light or quite black.
Regards.

**Comment by daeks** [comment]
New themes can be added to the whole IDE, have a look to the marketplace for existing ones or try to add an own one. (https://github.com/Codiad/Codiad/wiki/Themes)

One "old" light theme might be available https://github.com/daeks/Codiad-Theme-Clear but it is not further maintained

**Comment by nedark** [comment]
Hello. Thanks for the tip, I have installed the clear theme and now I am super-happy :D 
I didnt know about the existence of the clear theme - I suppose other users either.
Then I would change the suggestion to "iinclude clear theme by default" to allow every user to change color style with a simple options.
Regards

**Comment by daeks** [comment]
It is available in the marketplace and can be found there very easily.

**Comment by nedark** [comment]
I know. Found very easily != installed very easily. A lot of themes comes with codiad by default and selecting a light theme cause an awful layout. I think that (outer) clear theme should be loaded automatically when activating a light theme.
![image](https://cloud.githubusercontent.com/assets/8337317/6769476/878fefcc-d09d-11e4-88f6-c7648a762fed.png)
At least a warning about the risk of getting blind would be useful :D

**Comment by daeks** [comment]
Ah ok, now I unterstand the confusion. Thats because the "theme" you can select in the settings is only related to the editor itself and not related with codiad. Codiad is based on a thirdparty editor framework. 

The outer theme (which can be installed from the marketplace automatically) comes from Codiad.

**Comment by smolinari** [comment]
Yes, the overall IDE "theme" is a different animal than the workspace/ editor theme (ACE theme). The two should have different names. Like call the editor theme in the Settings just that, "Editor Theme" and call the IDE Theme in the Marketplace just that, "IDE Theme". Maybe add a short sentence underneath the editor setting like, "For overall different IDE Themes, check out the Marketplace!"

Scott

**Comment by daeks** [comment]
:+1:

**Comment by jeroen-net** [comment]
When I install the Clear theme, the theme doesn't show up in the list of themes. Is there something more that I have to do?

I would like to use Codiad in a classroom environment. A light theme would be helpful. Are there more light 'outer' / IDE themes?

**Comment by daeks** [comment]
It should be listed there automatically. Is it in the right place? Codiad searches for /themes/<yourfoldername>/theme.json

There might be more out there, but we only know that ones which are submitted to the marketplace. One example I also know from another issue is: https://github.com/sovrin/Codiad-Theme-Obsidian

**Comment by jeroen-net** [comment]
Hello Daeks,

Yes, the folder and the json-file are in the right place. Despite that the theme doesn't show up in the list. Thanks for pointing me to another theme. When I have time this summer, maybe I will make a new light theme.

**Comment by daeks** [comment]
Hi,
I will test it later as well.
New Themes are always welcomed :)
